hack/fill-empty-docs.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -e
+
+GOPATH="${GOPATH:-~/go}"
+export GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
+
+export PATH=$PATH:$GOPATH/bin
+export SUITES_PATH=cnf-tests/bin
+
+# generate the junit files
+hack/build-test-bin.sh
+cnf-tests/test-run.sh -junit _cache/junit -ginkgo.dryRun -ginkgo.v
```

Comment:
can you add the `_cache` in the gitignore?

Reply:
It's already there